### ISSUE TITLE
[$500] iOS - Inconsistent translation for accessibilityLabel of Floating Action Button

### ISSUE THREAD

**Issue by izarutskaya** [Expensify/App#26674]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Turn on "voice over" in your IOS device.
2. Long press or press Fab button.
3. Notice that it sounds "'New chat (Floating action)"
4. Change language to Spanish, long press or press Fab button.
5. Notice that it sounds "'Nuevo chat".
There's more simpler way to find it, we can search for `accessibilityLabel` in our en.js and es.js in out source code.

## Expected Result:
AccessibilityLabel of Fab button in Spanish should be "Nuevo chat (Acción flotante)"

## Actual Result:
AccessibilityLabel of Fab button in Spanish is "Nuevo chat" which is inconsistent with English.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [x] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** v1.3.56-18

**Reproducible in staging?:** Needs reproduction

**Reproducible in production?:** Needs reproduction

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

![Screenshot 2023-08-23 at 13 36 11](https://github.com/Expensify/App/assets/115492554/a33cc349-eb95-4e9a-8ce7-e6d89a4eccfb)
![Screenshot 2023-08-23 at 13 36 18](https://github.com/Expensify/App/assets/115492554/1f26c133-2b38-45c9-8983-68dc40ca949b)

**Expensify/Expensify Issue URL:**

**Issue reported by:** @hungvu193

**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1692772548251279

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~0190d81f792e377c25</li>
        <li>Upwork Job ID: 1701237346978205696</li>
        <li>Last Price Increase: 2023-09-25</li>
<li>Automatic offers: </li>
<ul>
<li>hungvu193 | Reporter | 26876528</li>
</ul></li>
    </ul>
</details>

**Comment by izarutskaya** [comment]
## Proposal from @hungvu193

### Please re-state the problem that we are trying to solve in this issue.
Inconsistent translation for accessibilityLabel of Floating Action Button
### What is the root cause of that problem?
We're using different word for `fabNewChat` with English and Spanish here, which make it inconsistency
https://github.com/Expensify/App/blob/4f412db5e26782740529eb0ef17b5db157628413/src/languages/en.js#L364
https://github.com/Expensify/App/blob/4f412db5e26782740529eb0ef17b5db157628413/src/languages/es.js#L363
### What changes do you think we should make in order to solve the problem?
We should change the translation of `fabNewChat` [here](https://github.com/Expensify/App/blob/4f412db5e26782740529eb0ef17b5db157628413/src/languages/es.js#L363) to:
`Nuevo chat (Acción flotante)`

### What alternative solutions did you explore? (Optional)
We can change the translation of `fabNewChat` [here](https://github.com/Expensify/App/blob/4f412db5e26782740529eb0ef17b5db157628413/src/languages/en.js#L364) to:
`New chat`

**Comment by slafortune** [comment]
Checking here -https://expensify.slack.com/archives/C049HHMV9SM/p1693943297038159?thread_ts=1692772548.251279&cid=C049HHMV9SM

Is it IOS that is translating this and not translating Chat since that is our name?

**Comment by slafortune** [comment]
Adding a new BZ member, I am out on sabbatical

**Comment by conorpendergrast** [comment]
I'm on parental leave; re-assigning! This has not been triaged at all, sorry Greg!

**Comment by greg-schroeder** [comment]
Awaiting proposal review

**Comment by greg-schroeder** [comment]
thoughts on proposals @abdulrahuman5196?

**Comment by greg-schroeder** [comment]
Bump @abdulrahuman5196

**Comment by greg-schroeder** [comment]
Re-adjusting C+ to someone with a bit more bandwidth!

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (First week)

**Comment by greg-schroeder** [comment]
bump @youssef-lr - can you please confirm the assignment here? 

> C+ reviewed @hungvu193's propsoal 🎀 👀 🎀
> 
> [#26674 (comment)](https://github.com/Expensify/App/issues/26674#issuecomment-1705391288)

**Comment by youssef-lr** [comment]
Thanks @greg-schroeder! Sorry folks, this is simple change and we'd like to keep internal.

**Comment by greg-schroeder** [comment]
Looks like we need to request a review on the PR eh @youssef-lr?

**Comment by youssef-lr** [comment]
Done & merged.
